test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -1,94 +1,88 @@
             }
         }
         It "Start-Job accepts arguments" {
-            $sb = { write-output $args[1]; write-output $args[0] }
+            $sb = { Write-Output $args[1]; Write-Output $args[0] }
             $j2 = Start-Job -ScriptBlock $sb -ArgumentList "$TestDrive", 42
-            Wait-job -TimeOut (5 * 60) $j2 | Should Be $j2
-            $r = receive-job $j2
-            $r -join "," | should be "42,$TestDrive"
+            Wait-job -Timeout (5 * 60) $j2 | Should Be $j2
+            $r = Receive-Job $j2
+            $r -Join "," | Should Be "42,$TestDrive"
         }
     }
     Context "jobs which take time" {
         BeforeEach {
-            $j = start-job -scriptblock { Start-Sleep 15 }
+            $j = Start-Job -ScriptBlock { Start-Sleep 15 }
         }
         AfterEach {
-            Get-Job | Remove-Job -force
+            Get-Job | Remove-Job -Force
         }
         It "Wait-Job will wait for a job" {
-            $result = wait-Job $j
-            $result | should be $j
-            $j.State | should be Completed
+            $result = Wait-Job $j
+            $result | Should Be $j
+            $j.State | Should Be "Completed"
         }
         It "Wait-Job will timeout waiting for a job" {
             $result = Wait-Job -Timeout 2 $j
-            $result | should be $null
+            $result | Should Be $null
         }
         It "Stop-Job will stop a job" {
-            Stop-Job -id $j.id
-            $j.State | should be Stopped
+            Stop-Job -Id $j.Id
+            $j.State | Should Be "Stopped"
         }
         It "Remove-Job will not remove a running job" {
             $id = $j.Id
-            remove-job $j -ErrorAction SilentlyContinue
+            Remove-Job $j -ErrorAction SilentlyContinue
             $job = Get-Job -Id $id
-            $job | should be $j
+            $job | Should Be $j
         }
         It "Remove-Job -Force will remove a running job" {
             $id = $j.Id
             Remove-Job $j -Force
             $job = Get-Job -Id $id -ErrorAction SilentlyContinue
-            $job | should be $null
+            $job | Should Be $null
         }
     }
     Context "Retrieving partial output from jobs" {
         BeforeAll {
-            function getResults($job, $n, $keep)
+            function GetResults($job, $n, $keep)
             {
                 $results = @()
 
-                # $count lets us bail out after 5 minutes, avoiding an endless loop
+                # $count allows us to bail out after 5 minutes, avoiding an endless loop
                 for ($count = 0; $results.Count -lt $n; $count++)
                 {
-                    if ($count -eq 10000)
+                    if ($count -eq 1000)
                     {
                         # It's been 5 minutes and we still have collected enough results!
                         throw "Receive-Job behaves suspiciously: Cannot receive $n results in 5 minutes."
```

Comment:
Yes, we should keep.  (This exception should raise _never_. So a performance should not be taken into account.)
The idea was that all the results of the tests have to be caught by `Should` that Pester  normally logs the results. But I check "Pester.xml" and see that the original throw is logged well.
Closed.